src/shared/label.ts

```diff
@@ -109,6 +109,6 @@ async function updateLabelColor(context: Context, labelName: string, color: stri
       color,
     });
   } catch (e: unknown) {
-    context.logger.error("Updating label color failed!", e);
+    context.logger.error("Updating label color failed!", { e });
```

Comment:
Seems very unusual that we have to wrap it in an object

Reply:
It seems that the accepted type works like that
https://github.com/ubiquity-os/ubiquity-os-logger/blob/development/src/types/log-types.ts#L18